test/e2e/usdc-swap-did/setup.sh

```
@@ -0,0 +1,278 @@
+if curl -s "${FAUCET_URL}/health" >/dev/null 2>&1; then
+    print_status "Faucet is running"
+else
+    print_warning "Faucet is not accessible. Tests may fail without funded accounts."
+fi
+
+echo ""
+
+# Step 3: Check IBC connections
+echo -e "${BLUE}Step 3: Checking IBC connections...${NC}"
+
+CONNECTIONS=$(snrd query ibc connection connections --node ${SONR_RPC} --output json 2>/dev/null | jq -r '.connections // [] | length')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)